DataManagementSystem/DB/FileCatalogDB.py

```diff
@@ -43,7 +22,17 @@ def __init__(self, databaseLocation='DataManagement/FileCatalogDB'):
     if db.find('/') == -1:
       db = 'DataManagement/' + db
 
-    DB.__init__(self, 'FileCatalogDB', db)
+    # DB.__init__(self, 'FileCatalogDB', db)
```

Comment:
Is it correct that the parents constructor is no longer called, or am I just not seeing its replacement?

Reply:
And it should use `super`

Reply:
Oups, this is a debugging artefact. Restored.